### DISCUSSION TITLE
win10使用mosdns，WIFI会断开一次，求解

### DISCUSSION THREAD

**Discussion by ghost** [IrineSistiana/mosdns#158] [Q&A]
不算什么严重问题，只是每一次电脑从关机、休眠、睡眠等状态恢复后，wifi在过个一两分钟的样子后会断开一次，重连后就稳定了。无论把微软相关几个域名的解释走socks5墙外解析，还是直接国内dns解析，都会这个样子。不走mosdns或用其他dns服务软件无此问题（用过greendns、AuroraGUI、Pcap_DNSProxy）。  

请问有办法解决这个问题吗?   
机器是 Surface pro，win10 21H1  

我设置调整的微软的几个相关域名如下：
```
microsoft.com
windows.com
msftncsi.com
msftconnecttest.com
```

**Comment by IrineSistiana** [comment]
神奇。mosdns 就是一个转发器。没有权限影响其他东西
我也一直在win里自用但没遇到这个问题。无法复现。
建议看看 debug log。
